app/models/authority.rb

```diff
@@ -176,6 +176,7 @@ def applications_per_week
     # Sunday is the beginning of the week (and the date returned here)
     # Have to compensate for MySQL which treats Monday as the beginning of the week
     h = by_week_from_sunday(applications, 'date_scraped')
+    return [] if h.empty?
```

Comment:
I actually think the comments_per_week version of this is much more readable. Do you think we should make this one consistent with that while we're here?